tests/events/kinesisFirehoseEvent.json

```diff
@@ -0,0 +1,31 @@
+{
+  "invocationId": "2b4d1ad9-2f48-94bd-a088-767c317e994a",
+  "deliveryStreamArn": "arn:aws:firehose:us-east-2:123456789012:deliverystream/delivery-stream-name",
+  "region": "us-east-2",
```

Comment:
We need to make two changes in this file:

1 - Rename it to `kinesisFirehoseKinesisEvent.json`
2 - Add the field `sourceKinesisStreamArn`

Reference: https://github.com/awslabs/aws-lambda-powertools-python/pull/1556/files#diff-b06a860206a2cce04772240219617495997d9ccd3faadaf4488b160e9abda0f1

Reply:
done